t/demo/demo.t

```diff
@@ -0,0 +1,23 @@
+use t::APISIX 'no_plan';
+
+log_level('debug');
+repeat_each(1);
+no_long_string();
+no_root_location();
+run_tests;
+
+__DATA__
+
+=== TEST 1: hello, world
+This is just a simple demonstration of the
+echo directive provided by ngx_http_echo_module.
+--- config
+    location = /t {
+        content_by_lua_block {
+            ngx.say('hello, world!')
+        }
+    }
+--- request
```

Comment:
Let's remove the duplicate sections from the beginning with https://github.com/apache/apisix/blob/7f9192576702eb3e5ab4764319756e9d91ab83c9/t/plugin/gzip.t#L33

Reply:
Thanks for point it out. fixed.